package.json

```diff
@@ -21,7 +21,6 @@
     },
     "devDependencies": {
         "@smooch/eslint-plugin-smooch": "1.0.3",
-        "babel": "6.5.2",
```

Comment:
I see that `babel` is used in the `npm run build` script above (line 10) so is `babel` not a devDependency still?  We advise customers to download the package from NPM and not be using the source code directly but they would need to have `babel` installed locally already in order to build this now, right?

Reply:
Actually the build is broken if you try it in the `v1-master` branch. It seems some logic was moved to babel-cli, according to the error I got after running `npm install` again to get all dependencies installed. So after running `npm install` this is what I got ```
You have mistakenly installed the `babel` package, which is a no-op in Babel 6.
Babel's CLI commands have been moved from the `babel` package to the `babel-cli` package.
```
Removing Babel, at the end resolved the build issue.